package.json

```diff
@@ -50,7 +50,7 @@
     "watchify": "^4.0.0",
     "webpack": "^5.76.0",
     "webpack-cli": "^4.9.1",
-    "wp-rocket-scripts": "^1.0.6",
+    "wp-rocket-scripts": "github:wp-media/rocket-scripts#fix/6959/ui-issue-lrc",
```

Comment:
I don't really like having a specific branch here as when it gets merged, the branch on `rocket-scripts` is likely to be deleted. 

Maybe you could create a sub-issue (new functionality of Github) to update WP Rocket and once the other one gets merged you'll be able to create a release on `rocket-scripts` and update here. What do you think ?